data/brands/amenity/fuel.json

```diff
@@ -49,6 +49,21 @@
         "name": "7-Eleven"
       }
     },
+    {
+      "displayName": "Mapco",
+      "locationSet": {
+        "include": ["001"],
```

Comment:
From my understanding, Mapco only has locations in the US. You should change the locationSet to include 'us' and remove the exclude argument.